ecllibrary/teststd/Date/TestDate.ecl

```diff
@@ -114,6 +114,35 @@ EXPORT TestDate := MODULE
     ASSERT(Date.MonthWeekNumFromDate(20180302, 6) = 2, CONST);  // Start of week = Friday
     ASSERT(Date.MonthWeekNumFromDate(20180302, 7) = 1, CONST);  // Start of week = Saturday
     ASSERT(Date.MonthWeekNumFromDate(20180303, 7) = 2, CONST);  // Start of week = Saturday
+    
+    ASSERT( Date.ISODayOfWeekFromDate(20140130) = 4, CONST);   // 4=Thursday
+	ASSERT( Date.ISODayOfWeekFromDate(19650217) = 3, CONST);   // 3=Wednesday
```

Comment:
Please vertically align the ASSERTS.  From this line onwards they appear to have two additional spaces.

Reply:
Fixed.